projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -74,6 +74,16 @@
   }
 }
 
+@mixin datagrid-action-button-styles($height: tokens.$cds-global-space-10) {
+  cursor: pointer;
+  width: 100%;
+  height: $height;
+  // Calculates how much padding to set on top and bottom by dividing the height by the icon size
+  padding: calc(($height - datagrid-variables.$clr-datagrid-icon-size) / 2)
+    calc((datagrid-variables.$clr-datagrid-fixed-column-size - datagrid-variables.$clr-datagrid-icon-size) / 2);
```

Check my suggestion. Extract what makes the most sense.

```suggestion
  // The vertical padding is half of the difference between height and icon size.
  // top and bottom are equiliteral.
  $vericalPadding =  calc(($height - datagrid-variables.$clr-datagrid-icon-size) / 2);
  
  // The horizontal padding is half of the difference between column width and icon size.
  // left and right are equiliteral.
  $horizontalPadding = calc((datagrid-variables.$clr-datagrid-fixed-column-size - datagrid-variables.$clr-datagrid-icon-size) / 2);
  
  padding: $vericalPadding $horizontalPadding;
```